go/libraries/doltcore/sqle/sqlfmt/row_fmt.go

```diff
@@ -113,10 +113,76 @@ func SqlRowAsInsertStmt(ctx context.Context, r sql.Row, tableName string, tableS
 	}
 
 	b.WriteString(")")
+	b.WriteString(" VALUES ")
 
-	b.WriteString(" VALUES (")
-	seenOne = false
+	// Write single insert
+	str, err := SqlRowAsSingleInsert(ctx, r, tableSch)
```

Comment:
(method comment, not this line)

This method should be a simple call of the two methods below, followed by a ";" to close the statement.